# see available mappings on src/bender_joy/xbox.py

# pause button
b_pause: 'START'

# cancel goal button
b_cancel: 'B'

# high priority joystick button
b_priority: 'LB'

# move neck up button
b_neck_up: 'UP'

# move neck down button
b_neck_down: 'DOWN'

# reset neck position button
b_neck_reset: 'RIGHT'

# movement axes
a_angular: 'LS_HORZ'
a_linear:  'LS_VERT'

# cmd velocities
max_linear_vel: 0.5   # caution: do not use values higher than 0.7 
max_angular_vel: 0.6  # caution: do not use values higher than 0.8

# neck angular velocity
max_neck_vel: 2
